citesphere/src/main/java/edu/asu/diging/citesphere/api/v1/user/ItemsApiController.java

```diff
@@ -106,5 +108,32 @@ public ResponseEntity<String> getCollectionsByGroupId(@RequestHeader HttpHeaders
         }
         return new ResponseEntity<String>(jsonResponse, HttpStatus.OK);
     }
+    
+    @RequestMapping(value = {"/user/citations"}, produces = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
the url should be consistent with the other ones; and since this is a search, it should have `search` in it (e.g. `items/search`)